libcrux/libcrux-platform/src/lib.rs

```diff
@@ -54,6 +54,8 @@ mod platform {
 
 #[cfg(not(hax))]
 mod platform {
+    use super::*;
```

Comment:
I am not sure why this is here. Did it fail before? Shouldn't CI have complained?

Reply:
Yeah, it gives an unused warning, but I ran into build issues when I removed it. I'll see if I can probably feature-gate that correctly.

Reply:
Yeah, usually that means that the thing using it is compiled conditionally